### ISSUE TITLE
Please, add italian dictionary to zettlr

### ISSUE THREAD

**Issue by teopost** [Zettlr/Zettlr#1153]
Please,
  add the italian to the default supported dictionary.

**Comment by canpolat** [comment]
@teopost With Zettlr yuo are not limited to the default installed dictionaries. You can always add dictionaries to Zettlr if you need them. Here is the documentation explaining how: [Add new dictionaries](https://docs.zettlr.com/en/reference/spell-checking/#add-new-dictionaries)

Hope this helps.

**Comment by teopost** [comment]
Yes. I read the documentation, but I hoped that the Italian dictionary was as important as those already supported.

**Comment by nathanlesage** [comment]
@teopost In fact, you cannot load the Italian dictionary at all, because of [this bug](https://github.com/wooorm/nspell/issues/11). You'd have to wait for [this issue](https://github.com/Zettlr/Zettlr/issues/1022) to be resolved, then hopefully we'll finally be able to support useful dictionary support (currently, German spellchecking is a joke just as well, so Italian speakers are not alone ...)
